scripts/nayduck.py

```diff
@@ -182,11 +182,16 @@ def run_locally(tests):
         # See nayduck specs at https://github.com/near/nayduck/blob/master/lib/testspec.py
         fields = test.split()
 
+        timeout = ""
         index = 1
+        ignored = []
         while len(fields) > index and fields[index].startswith('--'):
+            if fields[index].startswith("--timeout="):
+                timeout = fields[index].split("=")[1]
```

Comment:
For consistency let’s use single quotes for stings. For the second line I would have done it with `timeout = fields[index][10:]` but I guess split also works.